blockchain.go

```diff
@@ -931,8 +938,10 @@ func (b *Blockchain) ExecuteNextTransaction() (*types.TransactionResult, error)
 	blockContext := fvm.NewContextFromParent(
 		b.vmCtx,
 		fvm.WithBlockHeader(header),
-	)
-
+		fvm.WithReusableCadenceRuntimePool(
+			reusableRuntime.NewCustomReusableCadenceRuntimePool(1, func(runtime.Config) runtime.Runtime {
+				return b.cadenceRuntime
+			})))
```

Comment:
The code for building a new FVM context from a parent, given a header and the FVM options, is duplicated in `executeBlock` and `ExecuteScriptAtBlock`. Maybe refactor this into a function which is called in both places, so the FVM options stay consistent.

Reply:
this should be changed later on commits. `fvm.WithReusableCadenceRuntimePool` not needed anymore, I moved the debugger to initialization to global ( I had to actually, because `NewCustomReusableCadenceRuntimePool` loses config somehow, only way to use is `NewReusableCadenceRuntimePool` to keep debugger ) Long story short, I think it is not duplicated anymore there.